lnrpc/rpc.proto

```diff
@@ -1362,6 +1370,24 @@ message PendingChannelsResponse {
     repeated WaitingCloseChannel waiting_close_channels = 5 [ json_name = "waiting_close_channels" ];
 }
 
+message ChannelEventSubscription {
+}
+
+message ChannelEventUpdate {
+    oneof channel {
+        Channel open_channel = 1 [ json_name = "channel" ];
+        ChannelCloseSummary closed_channel = 2 [ json_name = "closed_channel" ];
+    }
+
+    enum UpdateType {
```

Comment:
As is, this PR is missing pending channels, and also newly opened channels. A channel can be open without actually being active. Please refer back to the original issue.

Reply:
Events for pending channels should be emitted from the funding manager. Events for opened channels should be emitted either at the funding manager or peer's `channelManager` goroutine.

Reply:
Also the following state transitions are missing:
  * open -> pending close -> closed

Reply:
discussed, pending ntfns will be in separate PR